shared-data/liquid-class/schemas/1.json

```
@@ -160,7 +160,7 @@
     },
     "correctionByVolume": {
       "type": "array",
-      "description": "Settings for volume correction keyed by target aspiration/dispense volume, representing additional volume the plunger should move to accurately hit target volume.",
+      "description": "Settings for volume correction, keyed by the total volume in tip after the aspirate/ dispense action. The ‘correction value’ represents an additional volume the plunger should move to accurately hit target volume.",
```

lol -- Andy insisted that the correction value is **NOT** an additional volume, and instead it's a convoluted ratio.

---

just some punctuation here…would remove the space after the slash and get rid of the curly quotes(!) around `correction value`.

---

> lol -- Andy insisted that the correction value is NOT an additional volume, and instead it's a convoluted ratio.

Ah, hmm..  then how about this-
```suggestion
      "description": "Settings for volume correction, keyed by the total volume in tip after the aspirate/dispense action. The correction value provides additional info to the plunger to accurately pipette target volume of the liquid class.",
```
But I think the sentence structure is a bit weird. Maybe @ecormany can help 😬

---

Hmm. "The correction value provides additional information about how much the pipette plunger should move to accurately pipette the specified volume when using this liquid class." ? 

IDK, it's not much better, and we don't need to fuss too much about schema descriptions.